### PR TITLE
Updated Images with container icons

### PR DESCRIPTION
**Pull request by ganesaar**
Images updated with container icons

### CODE REVIEW

**Review by suresh-ls — APPROVED** [review]

**Review by suresh-ls — COMMENTED** [review]
Can we add numbers to ML node in SWOP arch dia ?    Like  ML Node 1  & ML Node n

File: examples/mnist-pyt/MNIST-PYT.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The Machine Learning program, after conversion to Swarm Learning for the PyTorch
```
Comment (girisairam450) [inline examples/mnist-pyt/MNIST-PYT.md:14]: Images are enlarged and are not fitting to the screen. Can you fit the old width and height using the md file settings or resizing them?

Comment (girisairam450) [reply examples/mnist-pyt/MNIST-PYT.md:14]: Archana applied resizing changes in reverse proxy branch, So closing this comment,

**Review by girisairam450 — APPROVED** [review]
Reviewed

**Pull request merged**